http/handler.go

```diff
@@ -27,6 +27,9 @@ const (
 	// not to use request forwarding
 	NoRequestForwardingHeaderName = "X-Vault-No-Request-Forwarding"
 
+	// NoCORS is the name of the header telling Vault not to use CORS handler.
+	NoCORS = "X-Vault-No-CORS"
```

Comment:
What's the point of using this? If an admin set a policy to return CORS, why allow a client to override it?